pages/pipelines/rules.md

```diff
@@ -0,0 +1,169 @@
+# Pipeline rules
+
+Pipeline rules is a Buildkite feature used to manage and organize permissions between pipelines and other resources.
+
+Rules express that an action is allowed between a source resource (eg. a pipeline) and a target resource (eg. another pipeline). Rules allow you to break out of the defaults provided by Buildkite such as cluster boundaries.
+
+## Pipeline rules best practices
+
+### How should I use rules?
+
+Use rules to manage permissions between pipelines and other resources. Rules are typically used in tandem with [clusters](/docs/clusters/overview) to increase security and control, where clusters set hard boundaries and rules provide exceptions.
+
+### Available rules
+
+#### `pipeline.trigger_build.pipeline` 
+
+Cross-cluster pipeline triggering (eg. allow a pipeline in one cluster to trigger a pipeline in another cluster).
+
+Value fields:
+
+- `triggering_pipeline_uuid` The UUID of the pipeline that is allowed to trigger another pipeline.
+- `triggered_pipeline_uuid` The UUID of the pipeline that is allowed to be triggered by the `triggering_pipeline_uuid` pipeline.
+
+### `pipeline.artifacts_read.pipeline` Allowing jobs in a pipeline to access artifacts from builds in other pipelines.
+
+Value fields:
+
+- `source_pipeline_uuid` The UUID of the pipeline that is allowed to access artifacts from another pipeline.
+- `target_pipeline_uuid` The UUID of the pipeline that is allowed to have its artifacts accessed by jobs in the `source_pipeline_uuid` pipeline.
+
+### Example rule use
+
+Imagine you use two clusters to separate environments necessary for building your application (CI cluster) and deploying your application (CD cluster). Ordinarily pipelines in those clusters would not be able to trigger each other due to the isolation of clusters.
+
+Creating a `pipeline.trigger_build.pipeline` rule would allow triggering a pipeline in the CD cluster from a pipeline in the CD cluster. This would allow maintaining the separation of the CI and CD agents and still support triggering the deployment pipeline from the build pipeline.
```

Comment:
```suggestion
A `pipeline.trigger_build.pipeline` rule would allow a pipeline in the CI cluster to trigger a build for a pipeline in the CD cluster, while maintaining the separation of the CI and CD agents in their respective clusters.
```